src/main/java/io/logz/jmx2graphite/JavaAgentClient.java

```diff
@@ -0,0 +1,183 @@
+package io.logz.jmx2graphite;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Joiner;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.*;
+import javax.management.openmbean.CompositeData;
+import javax.management.openmbean.TabularData;
+import java.lang.management.ManagementFactory;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Created by roiravhon on 6/6/16.
+ */
+public class JavaAgentClient extends MBeanClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(JavaAgentClient.class);
+
+    MBeanServer server;
+    ObjectMapper objectMapper;
+
+    public JavaAgentClient() {
+
+        server = ManagementFactory.getPlatformMBeanServer();
+        objectMapper = new ObjectMapper();
+        objectMapper.setVisibility(PropertyAccessor.ALL, JsonAutoDetect.Visibility.NONE);
```

Comment:
Can you please comment in code why you did these two lines of setVisibility (I know, but I will forget)